tests/app/invite/test_invite_rest.py

```diff
@@ -190,7 +185,7 @@ def test_update_invited_user_for_wrong_service_returns_404(client, sample_invite
     assert json_response == "No result found"
 
 
-def test_update_invited_user_for_invalid_data_returns_400(client, sample_invited_user):
+def test_update_invited_user_for_invalid_data_returns_400(client, sample_invited_user, notify_db_session):
```

Comment:
Why is the `notify_db_session` necessary now? I think you alluded to it in a conversation last week..